torch/csrc/profiler/api.cpp

```diff
@@ -4,6 +4,16 @@ namespace torch {
 namespace profiler {
 namespace impl {
 
+ExperimentalConfig::ExperimentalConfig(
+    std::vector<std::string> profiler_metrics,
+    bool profiler_measure_per_kernel)
+    : profiler_metrics{profiler_metrics},
+      profiler_measure_per_kernel{profiler_measure_per_kernel} {}
+
+ExperimentalConfig::operator bool() const {
```

Comment:
`explicit operator bool()` would not be a safer way to do this?

Reply:
That's a great point.